support-frontend/assets/pages/supporter-plus-landing/setup/legacyActionCreators.ts

```diff
@@ -221,7 +220,7 @@ function regularPaymentRequestFromAuthorisation(
 				? billingState
 				: null,
 			// required Zuora field if country is US or CA
-			postCode: billingCountry === 'US' ? postCode : null,
+			postCode: null,
```

Comment:
Can we still provide it to Zuora/Stripe if it is given?

Reply:
Good point, will test that now on Code and re-enable. Thanks.

Reply:
Re: https://stripe.com/en-no/guides/optimizing-authorization-rates#:~:text=In%20particular%2C%20passing%20the%20ZIP,the%20customer%20and%20how%20much.

Reply:
Yes, tested  and this is back in place.